RemoteWellnessSupportApp/OnBoarding/OnboardingScreenView.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  OnboardingScreenView.swift
+//  RemoteWellnessSupportApp
+//
+//  Created by 岩本雄貴 on 2024/01/31.
+//
+
+import SwiftUI
+
+struct OnboardingScreenView: View {
+    @AppStorage("hasCompletedOnboarding") var hasCompletedOnboarding: Bool = false
+    @State private var selection = OnBoardingTab.introduction.rawValue
```

Comment:
Enumのrawvalueを使うと型の情報が失われるので、できる限りEnumのまま使うほうが望ましいです。

```suggestion
    @State private var selection = OnBoardingTab.introduction
```

Reply:
こちらで修正しました
https://github.com/y-iwamoto/RemoteWellnessSupportApp/pull/3/commits/b58902a5a00a82208cfb9d147e5a1c5500e2ec68